doc/modules/ensemble.rst

```diff
@@ -294,6 +294,12 @@ In practice those estimates are stored as an attribute named
 the value, the more important is the contribution of the matching feature
 to the prediction function.
 
+The impurity-based feature importance suffers from being computed 
+on statistics derived from the training dataset. 
```

Comment:
I think that the issue is not only about deriving statistics from the training set but the bias toward feature with the high-cardinality features.